unittests/auth_tests.cpp

```diff
@@ -17,81 +17,90 @@ using namespace eosio::testing;
 
 BOOST_AUTO_TEST_SUITE(auth_tests)
 
-BOOST_FIXTURE_TEST_CASE( missing_sigs, validating_tester ) { try {
-   create_accounts( {"alice"_n} );
-   produce_block();
+BOOST_AUTO_TEST_CASE_TEMPLATE( missing_sigs, TESTER, validating_testers ) { try {
+   TESTER chain;
```

Comment:
Would be nice if there was a way to combine `BOOST_AUTO_TEST_CASE_TEMPLATE` and `BOOST_FIXTURE_TEST_CASE` so that `TESTER chain;` was not needed. I don't see a way to do that, but maybe @spoonincode might know a way.

Reply:
I believe you can use the (sparsely documented?[^1]) `BOOST_FIXTURE_TEST_CASE_TEMPLATE` for example
```cpp
BOOST_AUTO_TEST_CASE_TEMPLATE( missing_sigs, T, validating_testers, T ) { try {
```
but there is a problem with dependent lookup so you're going to get stuck with doing either
```cpp
BOOST_AUTO_TEST_CASE_TEMPLATE( missing_sigs, T, validating_testers, T ) { try {
   this->create_accounts( {"alice"_n} );
   T::produce_block();
```
so it's not immediately clear to me if it's any better or not.

[^1]: I only found it documented at the very bottom of this page https://www.boost.org/doc/libs/1_85_0/libs/test/doc/html/boost_test/tests_organization/decorators/explicit_decorator_declaration.html

Reply:
Thanks for looking into this. Either way is fine. But I think `chain.create_accounts` is more readable. We can just keep it as is.